src/main/java/github/weichware10/analyse/gui/admin/Configurator.java

```diff
@@ -9,6 +9,8 @@
 import github.weichware10.util.config.Configuration;
 import github.weichware10.util.config.ZoomMapsConfiguration;
 import github.weichware10.util.gui.AbsScene;
+import github.weichware10.util.gui.Window;
+
 import java.io.File;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.io.File'